client/src/components/pages/__tests__/ChannelCreate.test.tsx

```diff
@@ -26,6 +43,40 @@ describe('Rendering', () => {
     expect(json).toBeTruthy();
     expect(json).toMatchSnapshot();
   });
+
+  it('renders [UserListItem]', async () => {
+    const user: AuthProviderMeQueryResponse['me'] | null = {
+      id: 'user-1',
+      verified: true,
+      email: 'test@email.com',
+      profile: null,
+    };
+
+    const component = createTestElement(
+      <AuthContextProvider
+        value={{
+          user,
+          setUser: jest.fn(),
+          signOutAsync: jest.fn(),
+          loadMeQuery: jest.fn(),
+        }}>
```

Comment:
I think it is cleaner to mock `useAuthContext` than overriding values here. See my other comment in `providers/AuthProvider.tsx` file.

Reply:
give me some time for me to try as suggested.. because I'm not used to `mocking` yet :)

Reply:
I made new commit and hope it meets your expectation :)
https://github.com/dooboolab/hackatalk/pull/474/commits/1f9bb6e774a7054b177caca84da4bc9ec485fcbd